<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewTest</class>
 <widget class="QMainWindow" name="NewTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NewTest</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(146, 208, 80);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_StartTesting">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>430</y>
      <width>189</width>
      <height>78</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(56, 87, 35);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Start
Testing</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Return">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>430</y>
      <width>189</width>
      <height>78</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(56, 87, 35);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Return to
Main Menu</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Files">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>250</y>
      <width>691</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentText">
     <string/>
    </property>
    <property name="placeholderText">
     <string>File Directory</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Filename">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>340</y>
      <width>691</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Default Filename</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Rename">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>430</y>
      <width>189</width>
      <height>78</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(56, 87, 35);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Rename
Filename</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>40</y>
      <width>191</width>
      <height>191</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../assets/broccoli.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Exit">
    <property name="geometry">
     <rect>
      <x>987</x>
      <y>-3</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Minimize">
    <property name="geometry">
     <rect>
      <x>946</x>
      <y>-3</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>_</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
